prombench/manifests/cluster_gke.yaml

```diff
@@ -3,7 +3,7 @@ zone: {{ .ZONE }}
 cluster:
   name: {{ .CLUSTER_NAME }}
   
-  initialclusterversion: 1.25
+  initialclusterversion: 1.26
```

Comment:
bumped this to `1.26` because i kept running into this error

```
14:44:14 gke.go:222: Couldn't create cluster 'prombench', file:manifests/cluster_gke.yaml ,err: rpc error: code = InvalidArgument desc = No valid versions with the prefix "1.25" found.
error details: name = RequestInfo id = 0x31762f67e3d72c90 data = 
error details: name = DebugInfo detail = INVALID_ARGUMENT: no valid versions with the prefix "1.25" found stack = cloud/kubernetes/engine/common/error_desc.go:458 +0x26 google3/cloud/kubernetes/engine/common/errdesc.(*GKEErrorDescriptor).createErr(0xc00196abc0, {0x5557fcce0818, 0x555806731460}) cloud/kubernetes/engine/common/error_desc.go:316 +0x4c google3/cloud/kubernetes/engine/common/errdesc.(*GKEErrorDescriptor).WithMsgCtx(0xc0549236d8?, {0x5557fcce0818?, 0x555806731460?}, {0x5557efbed73b, 0x2a}, {0xc06d63e0c8, 0x1, 0x1}) cloud/kubernetes/engine/common/error_desc.go:308 google3/cloud/kubernetes/engine/common/errdesc.(*GKEErrorDescriptor).WithMsg(...) cloud/kubernetes/engine/common/version_alias.go:83 +0x4ad google3/cloud/kubernetes/engine/common/version.DealiasVersionStringFull({0xc0549236d8, 0x4}, {{0x1, 0x1d, 0x4, 0xfea3a}, {0xc00370127f, 0xb}}, 0xc07bbf2ed0, 0x30?) cloud/kubernetes/engine/common/version_alias.go:44 +0x3b google3/cloud/kubernetes/engine/common/version.DealiasVersionString({0xc0549236d8?, 0x0?}, {{0x1, 0x1d, 0x4, 0xfea3a}, {0xc00370127f, 0xb}}, 0x5557f13bd674?, 0x88?) cloud/kubernetes/server/patch/field/master/cluster_master_version.go:79 +0x268 google3/cloud/kubernetes/server/patch/field/master/clusterversion.(*masterVersionValidator).ApplyDefaults(0xc03c43be10, {0x5557fcce0738, 0xc09e868a80}, 0xc07fd2d560) cloud/kubernetes/server/patch/common/field.go:113 +0x56 google3/cloud/kubernetes/server/patch/common/patchbase.(*Field).ApplyDefaults(0xc057745c80, {0x5557fcce0738, 0xc09e868a80}, 0xc07fd2d560) cloud/kubernetes/server/patch/patcher/field_list.go:221 +0x15c google3/cloud/kubernetes/server/patch/patcher/fieldlist.FieldList.ApplyDefaults({0xc06d602408, 0x9c, 0xc099f7f2c0?}, {0x5557fcce0738, 0xc09e868a80}, 0xc07fd2d560) cloud/kubernetes/server/patch/common/field_interfaces.go:150 +0x227 google3/cloud/kubernetes/server/patch/common/patchbase.MergeAndDefault({0x5557fcce0738, 0xc09e868a80}, 0x5557fc0fa960?, 0xc07fd2d560, 0xc06d63ead8?, 0x5557f6175cde?, {0x5557fcbc4bc0?, 0xc063946b08?}, {0x5557fcbc4bc0, 0xc06d489608}) cloud/kubernetes/server/patch/patcher/cluster_fields.go:381 +0x7e google3/cloud/kubernetes/server/patch/patcher/cluster.(*patcher).Merge(0xc06d63eb18?, {0x5557fcce0738?, 0xc09e868a80?}, 0x1?, 0xc064baf110?)
make: *** [Makefile:16: cluster_create] Error 1
```